.github/workflows/build.yml

```diff
@@ -123,7 +123,6 @@ jobs:
       with:
         name: pulumi-${{ env.PROVIDER }}-provider.tar.gz
         path: ${{ github.workspace }}/bin/provider.tar.gz
-        retention-days: 30
```

Comment:
Hoping this here doesn't matter too much but can upstream to ci-mgmt if needed. cc @pulumi/providers